applications/nrf_desktop/src/modules/Kconfig.caf_settings_loader.default

```diff
@@ -10,32 +10,55 @@ config DESKTOP_SETTINGS_LOADER
 	bool "Settings loader module (CAF)"
 	select CAF_SETTINGS_LOADER
 	select SETTINGS
-	imply NVS
+	imply ZMS if SOC_FLASH_NRF_RRAM
+	imply NVS if !SOC_FLASH_NRF_RRAM
 	imply FLASH
 	imply FLASH_MAP
 	imply FLASH_PAGE_LAYOUT
 	help
 	  nRF Desktop uses settings loader module from Common Application
 	  Framework (CAF).
 
-	  By default, nRF Desktop application uses non-volatile storage backend
-	  and the settings partition is located in internal FLASH.
+	  By default, nRF Desktop application, depending on the non-volatile memory
+	  technology used by the device, uses either the Zephyr Memory Storage (ZMS)
+	  or Non-Volatile Storage (NVS) backend.
+	  ZMS is used for the devices with the RRAM non-volatile memory that do not
+	  require explicit flash erases.
+	  Otherwise, NVS is used.
```

Comment:
Can we make it as a single paragraph? Generally `Kconfig help` style suggests only 80 character per line limitation (instead of one sentence per line)